website1/src/scss/sections/_contacts.scss

```
@@ -0,0 +1,226 @@
+.contacts {
+  &-wrapper {
+    width: 100%;
+    background-color: $white-back;
+  }
+
+  &__title {
+    margin-bottom: 68px;
+    position: relative;
+  }
+
+  &__callback {
+    position: absolute;
+    top: 156px;
+    left: 515px;
+    max-width: 849px;
+    height: 696px;
+    padding: 52px 70px 69px 80px;
+    background: $white;
+    z-index: $one;
+    box-shadow: 0 7px 24px rgba(164, 151, 120, 0.17);
+
+    &-title {
+      font-size: 24px;
+      line-height: 127%;
+      text-transform: uppercase;
+      margin-bottom: 15px;
+    }
+
+    &-subtitle {
+      max-width: 371px;
+      font-weight: 300;
+      line-height: 23px;
+      margin-bottom: 53px;
+    }
+
+    &-form {
+      font-weight: 300;
+      line-height: 22px;
+      height: 60px;
+      padding-left: 30px;
+      font-size: 18px;
+      border: 1px solid #DDDDDD;
+      margin-bottom: 16px;
+    }
+
+    &-name-phone {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    &-name {
+      width: 48%;
+      min-width: 147px;
+    }
+
+    &-phone {
+      width: 48%;
+      min-width: 147px;
+    }
+
+    &-mail {
+      width: 100%;
+    }
+
+    &-text {
+      width: 100%;
+      min-height: 201px;
+      padding-top: 27px;
+      font-family: Source Sans Pro;
```

Add quotes.
```suggestion
      font-family: "Source Sans Pro";
```
[font-family](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family)